libs/ui/src/lib/breadcrumbs/styled.ts

```diff
@@ -7,9 +7,9 @@ const StyledBreadcrumbs = styled.nav`
   flex-direction: column;
```

Comment:
Migrere til css-modules?

Reply:
God innspill. Opreretter et issue på det, for vi har mye style-components som ikke er css-modules. Kanskje vi kan ta alt i samme sleng.
issue https://github.com/Informasjonsforvaltning/catalog-frontend/issues/62